models/meshmodel/registry/registry.go

```diff
@@ -242,7 +245,14 @@ func (rm *RegistryManager) GetModels(db *database.Handler, f types.Filter) ([]v1
 		fmt.Println(modelWithCategoriess)
 		fmt.Println(err.Error()) //for debugging
 	}
-	for _, modelDB := range modelWithCategoriess {
+
+	for num, modelDB := range modelWithCategoriess {
+		filter := &v1alpha1.ComponentFilter{}
+		entities, _, _ := rm.GetEntities(filter)
+		host := rm.GetRegistrant(entities[num])
+		modelDB.HostID = host.ID
+		modelDB.HostName = host.Hostname
+		modelDB.DisplayHostName = HostnameToPascalCase(host.Hostname)
```

Comment:
FormatAsReadableString doesn’t serve the purpose completely right?

Reply:
Yes